packages/medusa/src/api/middlewares/order/apply-order-includes-options.ts

```diff
@@ -0,0 +1,35 @@
+import { NextFunction, Request, Response } from "express"
+import { FindConfig } from "../../../types/common"
+import { Order } from "../../../models"
+
+const INCLUDES_FIELDS = ["returnable_items"] as const
+
+/**
+ * Retrieve the includes options from the select prop of retrieveConfig.
+ * If the include option is present then assigned it to includes on req
+ * @param req
+ * @param res
+ * @param next
+ */
+export function applyOrderIncludesOptions(
```

Comment:
@olivermrbl here is the proposed middleware

Reply:
comment: I like the idea. As you've alright highlighted, this is unnecessarily coupled with Orders. The `includes` fields can be defined in the `index.ts` of domains and passed to the middleware when registering it. Similar to our transformers.

Reply:
I will proceed with the changes then 👍